indra/resources/update_resources.py

```diff
@@ -774,6 +774,40 @@ def update_biomappings():
                       delimiter='\t')
 
 
+def update_lspci():
+    # We first create a dict of LSPCIs and their members but only for ones
+    # that actually have TAS statements corresponding to them
+    from indra.sources import tas
+    tp = tas.process_from_web(affinity_class_limit=10)
+    lspci_members = defaultdict(set)
+    for stmt in tp.statements:
+        if 'LSPCI'not in stmt.subj.db_refs:
+            continue
+        for k, v in stmt.subj.db_refs.items():
+            if k in {'TEXT', 'LSPCI'}:
+                continue
+            lspci_members[stmt.subj.db_refs.get('LSPCI')].add((k, v))
+
+    # We then process the names table in a way that we always prioritize the
+    # first row for each LSPCI since the table is pre-sorted by priority
+    df = pandas.read_csv('lsp_compound_names.csv')
```

Comment:
```suggestion
    df = pandas.read_csv('lsp_compound_names.csv', dtype={'lspci_id': str})
```

this means you don't have to keep casting